cookbook/doctrine/mongodb_session_storage.rst

```diff
@@ -162,7 +162,7 @@ Because MongoDB uses dynamic collection schemas, you do not need to do anything
 session collection. However, you may want to add an index to improve garbage collection performance.
 From the `MongoDB shell`_:
 
-.. code-block:: sql
+.. code-block:: text
```

Comment:
It even isn't shell code, is it? I mean it's the MongoDB shell, but I am not sure if the bash highligher will handle that properly. Let's try it (see #6192).